packages/app/src/app/overmind/actions.ts

```diff
@@ -94,15 +94,32 @@ export const modalClosed: Action = ({ state }) => {
   state.currentModal = null;
 };
 
-export const signInClicked: AsyncAction<{ useExtraScopes: boolean }> = (
-  { actions },
-  options
-) => actions.internal.signIn(options);
+export const signInClicked: Action<string | void> = ({ state }, redirectTo) => {
```

Comment:
You could just use a default value too I think

```suggestion
export const signInClicked: Action<string | void> = ({ state }, redirectTo = '') => {
```

Reply:
Done